src/screens/Overview/components/Stats/index.js

```diff
@@ -46,15 +54,20 @@ export default ({instructor, revenue}) => {
                 </div>
               : null
             }
-            <RevenuePeriod
-              title={`Last ${size(revenue)} months`}
-              revenue={currentTotalRevenue.revenue}
-              subscriberMinutes={currentTotalRevenue.minutes_watched}
-            />
-          </div>
-        : <div className='mt3 green'>
-            {noRevenueDescriptionText}
+            {currentTotalRevenue && currentTotalRevenue.revenue > 0
+              ? <RevenuePeriod
+                  title={`Last ${currentTotalRevenue.monthCount} months`}
```

Comment:
Good idea. Will be resolved when we upgrade to a proper localization library. Until then shouldn't be an issue as we will show more than 1 month.